src/router.dispatch.ts

```diff
@@ -23,16 +24,20 @@ const routerDispatch = (app: Application) => {
     router.use(Urls.HEALTHCHECK, HealthCheckRouter);
     router.use(featureFlagMiddleware);
     router.use(localeMiddleware);
-    router.use(Urls.HOME, HomeRouter);
-    router.use(Urls.BEFORE_YOU_FILE_PACKAGE_ACCOUNTS, BeforeYouFilePackageAccountsRouter);
+
+    router.use(Urls.HOME, extractCompanyNumberMiddleware, HomeRouter);
```

Comment:
If we know the url here doesn't include the company number, why are we using the extract middleware?